reconcile/utils/terrascript_aws_client.py

```diff
@@ -299,6 +300,13 @@ class UnapprovedSecretPathError(Exception):
     pass
 
 
+class Moved(Block):
+    """Terraform `moved` block, available since Terraform 1.1"""
+
+    def __init__(self, fro: str, to: str):
+        super().__init__(fro=fro, to=to)
```

Comment:
nit: why not?

```suggestion
    def __init__(self, from: str, to: str):
        super().__init__(from=from, to=to)
```

Reply:
`from` being a python keyword, this seems unfortunately not possible

Reply:
very true :(